engine/access/rpc/backend/backend_scripts.go

```diff
@@ -90,18 +93,27 @@ func (b *backendScripts) executeScriptOnExecutionNode(
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to find execution nodes at blockId %v: %v", blockID.String(), err)
 	}
+	// encode to MD5 as low compute/memory lookup key
+	encodedScript := md5.Sum(script) //nolint:gosec
```

Comment:
curious why the `//nolint:gosec` was needed. What was the linter complaining about?

Reply:
`G401: Use of weak cryptographic primitive (gosec)` was the warning, but we're not using it for cryptographic purposes, so it's safe to ignore

Reply:
got it ..thanks